web/src/components/View/View.css

```diff
@@ -1,39 +1,31 @@
 .view-container--inner {
-    display: flex;
     justify-content: center;
     flex-direction: column;
     font-family: var(--colarkfont);
     width: 80%;
     margin: 0 auto;
-    height: 400px;
+    color: var(--colarkfontcolor);
 }
 
 
 .view-container__title {
-    color: var(--colarkblue);
     text-align: left;
     font-size: 48px;
     font-weight: 400;
     letter-spacing: 6px;
     width: 100%;
-    margin-top: 30px;
-}
-
-.view-container__subtitle {
-    display: flex;
-    justify-content: left;
-    font-size: 24px;
-    font-weight: 400;
-    color: var(--lightgray);
-    width: 100%;
-    letter-spacing: 3px;
-    margin-bottom: 30px;
+    margin-top: 70px;
+    height: 300px;
 }
 
 @media only screen and (max-width: 600px) {
     .view-container--inner {
         margin-top: 105px;
     }
+
```

Comment:
Is this preventing the View text from moving around when the window width changes?  Now that the old illustration is gone, there will be no need for it to move; we just have to make sure it looks good in mobile.

Reply:
ya this is for mobile so that it doesn't squish up too close to the navbar - i changed it to padding in the new nabar PR that's next.